src/explore-education-statistics-frontend/src/modules/table-tool/components/__tests__/__data__/tableData.ts

```diff
@@ -0,0 +1,313 @@
+import {
+  CategoryFilter,
+  Indicator,
+  LocationFilter,
+  TimePeriodFilter,
+} from '@common/modules/table-tool/types/filters';
+import { TableHeadersConfig } from '@common/modules/table-tool/types/tableHeaders';
+import { FullTable } from '@common/modules/table-tool/types/fullTable';
+import { TableDataQuery } from '@common/services/tableBuilderService';
+
+// eslint-disable-next-line no-shadow
+enum ReleaseType {
+  AdHoc = 'Ad Hoc',
+  NationalStatistics = 'National Statistics',
+  OfficialStatistics = 'Official Statistics',
+}
```

Comment:
Don't think we need this? (it also already exists in `@common/services/publicationService`)